proto/desmos/profiles/v1beta1/models.proto

```diff
@@ -92,15 +94,48 @@ message UserBlock {
   option (gogoproto.equal) = true;
   option (gogoproto.goproto_stringer) = true;
 
-  string blocker = 1 [(gogoproto.moretags) = "yaml:\"blocker\""];
-  string blocked = 2 [(gogoproto.moretags) = "yaml:\"blocked\""];
+  string blocker = 1 [ (gogoproto.moretags) = "yaml:\"blocker\"" ];
+  string blocked = 2 [ (gogoproto.moretags) = "yaml:\"blocked\"" ];
   string reason = 3 [
     (gogoproto.moretags) = "yaml:\"reason\""
   ]; // Reason the user has been blocked for (optional).
-  string subspace = 4 [(gogoproto.moretags) = "yaml:\"subspace\""];
+  string subspace = 4 [ (gogoproto.moretags) = "yaml:\"subspace\"" ];
 }
 
 // UserBlocks wraps a list of UserBlock objects
 message UserBlocks {
-  repeated UserBlock blocks = 1 [(gogoproto.nullable) = false];
-}
\ No newline at end of file
+  repeated UserBlock blocks = 1 [ (gogoproto.nullable) = false ];
+}
+
+// ___________________________________________________________________________________________________________________
+
+// Proof contains all the data used to verify the signature when linking an
+// account to a profile
+message Proof {
```

Comment:
I think we should add an additional fields here named `value` that represents the plain text value that has been signed with the private key in order to produce the signature. Otherwise, without it we will not be able to check the validity of the signature

Reply:
@RiccardoM why not naming it `plain_text` directly?

Reply:
I agree with @bragaz, naming it `plain_text` would be more explicit.

Reply:
@bragaz @dadamu Yeah that's perfect